v2/cf-nodejs-logging-support/src/lib/middleware/middleware.ts

```diff
@@ -2,16 +2,25 @@ import LevelUtils from "../logger/level-utils";
 import RecordFactory from "../logger/record-factory";
 import ResponseAccessor from "./response-accessor";
 import RootLogger from "../logger/root-logger";
+import { JWTService } from "../config/jwt-service";
+import RequestAccessor from "./request-Accessor";
 
 export default class Middleware {
 
     static logNetwork(req: any, res: any, next?: any) {
         let logSent = false;
 
-        const networkLogger = RootLogger.getInstance().createLogger();
+        let networkLogger = RootLogger.getInstance().createLogger();
         networkLogger.initContext(req);
+
+        var dynLogLevelHeader = JWTService.getDynLogLevelHeaderName();
+        var token = RequestAccessor.getInstance().getHeaderField(req, dynLogLevelHeader);
+        if (token) {
+            networkLogger = JWTService.bindDynLogLevel(token, networkLogger);
```

Comment:
It is not required to overwrite networkLogger. It is the same object that is also passed as argument.